src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String name;
+    private int quantity;
```

Comment:
It's good practice to include a default constructor in your model classes to provide flexibility for object creation, especially when working with frameworks that rely on them.

Reply:
++